TelosLabs/giving-connection
# Feature | Change search pill behavior from OR to AND

### Context
Initially, the queries to filter by Causes, Services and Populations Served were designed with OR format. For example, when filtering for three different causes, the results will include organizations that have at least one. Now the search format needs to be migrated to AND, using the same example above, the results will only include organizations that have all three causes.

### What changed
Modified Causes, Services, and Population Served queries.

### How to test it
1. Add causes, services and populations served, listed on the search pills, to an organization.
2. Filter by more than one cause, every time a cause is added to the search, the results should decrease. Only locations that have all causes will appear in the results. 
3. Applied step 2 with Services and Populations served.

### References

[ClickUp ticket](https://app.clickup.com/t/85yx52u3n)
